### ISSUE TITLE
[Bug] Can not load a local directory repo via browsing

### ISSUE THREAD

**Issue by NiceFeather** [orgzly/orgzly-android#634]
- [x] I have searched for existing issues that may be the same as or related to mine.

I have a webDAV repo and working fine, but I can not load a local directory repo via browsing, the position of the repo is actually at "/stage/emulated/0/Backup/Data/".

Device: Samsung s7 edge
OS: Android 8.0
Orgzly: 1.8.2

![SmartSelect_20191114-163929_Orgzly](https://user-images.githubusercontent.com/28109556/68842093-be296600-0700-11ea-9962-bd3a346564a5.jpg)
![SmartSelect_20191114-164407_Orgzly](https://user-images.githubusercontent.com/28109556/68842321-2710de00-0701-11ea-999e-8f4486daaeb4.jpg)

**Comment by nevenz** [comment]
> I can not load a local directory repo via browsing, the position of the repo is actually at "/storage/emulated/0/Backup/Data/".

What do you mean you cannot? Do you get an error when you click the "Browse" button, or does the directory not show up when browsing?

Or are you expecting a different URI? That URI looks like a valid content provider URI to me (which is preferable to use over "file:" URI and might have to become the only option once app starts targetting Android 10).

**Comment by NiceFeather** [comment]
> What do you mean you cannot? Do you get an error when you click the "Browse" button, or does the directory not show up when browsing?

Yes, there was no any error while browsing directory, but returned the content as the screenshot attached and failed loading.

BTW, I have confirmed that some org files are located in"/storage/emulated/0/Backup/Data/"

**Comment by nevenz** [comment]
> Yes, there was no any error while browsing directory, but returned the content as the screenshot attached and failed loading.

What's the error message?

Note that you cannot currently have files with the same name in two or more repositories that you're using.

**Comment by NiceFeather** [comment]
So sorry heard that :)
Thanks for your note, that is the case. In order to do the test, I copied some files with the same name as webDAV and placed it locally. there was without any error messages, org just cannot be loaded and displayed.

But this why? i mean have files with the same name in two or more repositories

**Comment by nevenz** [comment]
> But this why? i mean have files with the same name in two or more repositories

Notebook name currently has to be unique in the database. That needs fixing. #225 is another good reason to do it.
